crashlogger.js

```diff
@@ -8,16 +8,29 @@
  * @license MIT license
  */
 
+/// <reference types="nodemailer" />
```

Comment:
It will be eventually so `transport` has its proper type, but for now, not so much. `tsc` will complain about the `nodemailer` module not existing though.